010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
+**Safe**: Postgres comes with ACID transactions and its transaction isolation levels are [well documented](https://www.postgresql.org/docs/current/transaction-iso.html), making it easy to create correct applications that are easy to reason about. It also provides a lot of mechanisms for data integrity, including constraints, triggers, and foreign keys.
 
-A Serverless Data Platform, on the other hand, is focused on adding data-related functionality, like free-text-search, advanced aggregations, caching, block storage, and so on. It doesn’t provide things like hosting, authentication, or push notifications, but it integrates nicely with platforms like Vercel, Netlify, or Cloudflare pages, as well as other authentication providers.
+**Performant**: Postgres transactions are implemented via a multi-version concurrency control (MVCC) system, which means it is very good at handling concurrent writes. The Postgres query planner is sophisticated and can handle complex queries efficiently.
 
-![BaaS versus Serverless Data Platform](images/baas-vs-sdp.png)
+**No vendor lock-in**: PostgreSQL is offered as a managed service by many cloud providers and it is open-source, so you can also always self-host if you choose to. In addition to this, the Postgres wire protocol is open and well-documented, and because of this it was adopted by other database systems as well (e.g. CockroachDB and YugabyteDB). This means that if your application depends on PostgreSQL, you are guaranteed to have a lot of options when it comes to hosting and scaling.
 
-Because BaaS typically provide built-in authentication and row-level-security rules, it is possible to write strictly the client side of the application, and write no backend code at all (hence the name). This can be great for small projects because they get everything from a single provider, but as the projects grow in requirements and scale, some serious challenges appear:
+**Versatile**: Besides the usual relational database features, Postgres also has a rich set of data types, including JSON and geospacial data (via the PostGis extension). It also has row-level security, inverted indexes (GIN), and many other features that make it a good fit for a wide range of applications.
 
-- security rules become more complex and they are better implemented and maintained in code
-- each sub-service in the BaaS (hosting, authentication) is relatively shallow compared to dedicated providers
-- there is significant functionality overlap between what the BaaS offers and what modern full-stack web frameworks offer
+**Extensible**: Postgres has a rich ecosystem of extensions and new extensions can be written in multiple programming languages, including C, [Rust](https://github.com/pgcentralfoundation/pgrx), and [Zig](https://github.com/xataio/pgzx) (pgzx is maintained by Xata). New extensions can add new data types, new indexing methods, new query operators, and new procedural languages.
```

Comment:
It wasn't obvious to me that Zig linked to pgzx. I came in to add a URL for pgzx when it was referenced.